### ISSUE TITLE
Testing MIoU is Nan in cityscapes dataset (1024x520)

### ISSUE THREAD

**Issue by herleeyandi** [davidtvs/PyTorch-ENet#13]
Hello @davidtvs I follow the way how to train the cityscapes. The validation's Mean IoU is 0.5426, Then I try to do testing however I got nan result, it said because of  `Mean of empty slice`. Here is the full problem.
```
Testing...

>>>> Running test dataset
/home/my_home/PyTorch-ENet/metric/iou.py:93: RuntimeWarning: Mean of empty slice
  return iou, np.nanmean(iou)
>>>> Avg. loss: 0.0000 | Mean IoU: nan
unlabeled: nan
road: nan
sidewalk: nan
building: nan
wall: nan
fence: nan
pole: nan
traffic_light: nan
traffic_sign: nan
vegetation: nan
terrain: nan
sky: nan
person: nan
rider: nan
car: nan
truck: nan
bus: nan
train: nan
motorcycle: nan
bicycle: nan
```

**Comment by davidtvs** [comment]
See issue #8. Test set for Cityscapes is not public
